data/fixtures/recorded/actions/splitDefineVest.yml

```diff
@@ -0,0 +1,39 @@
+languageId: typescript
+command:
+  version: 1
+  spokenForm: split define vest
+  action: executeCommand
```

Comment:
This doesn't look right. Are you sure this wasn't recorded using your personal version from custom actions csv?

Reply:
`version: 1`? I assume you just copied an existing file instead of recording a new one?

Reply:
Yes, I copied the existing file because I wasn't able to figure out how to run the command from the debug editor-the test was passing so I was pretty sure the command was working, though.

Reply:
This doesn't actually test any of your code, though

Reply:
Ok, then I misunderstood how it works. I don't really understand how to do the local recording for test cases yet.

Reply:
Assuming you found the docs for the test recorder?

Reply:
I did. Will take a look when I have some more time. Thanks!